src/sql/workbench/contrib/query/browser/queryResultsEditor.ts

```diff
@@ -76,8 +76,8 @@ export class QueryResultsEditor extends EditorPane {
 	public static ID: string = 'workbench.editor.queryResultsEditor';
 	protected _rawOptions: BareResultsGridInfo;
 
-	private resultsView: QueryResultsView;
-	private styleSheet = DOM.createStyleSheet();
+	private resultsView?: QueryResultsView;
```

Comment:
Results view is technically guaranteed to exist as well since if createEditor is called that starts existing. So you can just use ! for that too